marathon_lb.py

```diff
@@ -550,6 +637,7 @@ def config(apps, groups, bind_http_https, ssl_certs, templater,
 
         do_backend_healthcheck_options_once = True
         key_func = attrgetter('host', 'port')
+        taken_server_ids = set()
```

Comment:
would it make more sense to use dict()? or other similar data structure for fast lookup